prime-router/src/main/kotlin/Element.kt

```diff
@@ -759,6 +766,72 @@ data class Element(
         return codeValue?.code
     }
 
+    /**
+     * Determines if an element needs to use a mapper given the [elementValue].
+     * @return true if a mapper needs to be run
+     */
+    fun useMapper(elementValue: String): Boolean {
+        val overrideValue = mapperOverridesValue != null && mapperOverridesValue
+        return mapperRef != null && (overrideValue || elementValue.isBlank())
```

Comment:
it would be weird if the mapperOverrides flag is true but there is no mapper.

Moving to the "mappers always run always" philosophy would help clean this up, but this is a step in that direction.